src/player.cpp

```diff
@@ -6445,6 +6447,9 @@ void player::hardcoded_effects(effect &it)
             }
         }
     } else if( id == effect_bleed ) {
+        // Prolonged haemorrhage is a significant risk for developing anaemia
+        vitamin_mod( vitamin_iron, rng( -1, -4 ) );
```

Comment:
Yes every turn, I haven't been following the vitamin code long enough to understand how significant that 180 (average 75) points of iron deficiency would be, but bleeding for 30 pts of damage is certainly something that can happen.  The other implication is that you are losing iron even if you aren't losing blood (because you're applying pressure to the wound while treating it).